BBBSWA-38 Creating the footer component

velvetarchangel opened the pull request:
Created the footer component so it is reuseable for every page we will
use. Also created different vues for applicant administrator and
Admindashboard.

velvetarchangel committed:
BBBSWA-38 Creating the footer component

Created the footer component so it is reuseable for every page we will
use. Also created different vues for applicant administrator and
Admindashboard.

ckthiessen commented:
The text on the right is not in-line with the text on the left. It should be vertically centered in the div like the buttons are. 
![image](https://user-images.githubusercontent.com/26128858/99158051-cf96ac80-268b-11eb-9df6-35b3cf6d5afe.png)

ckthiessen commented on frontEnd/src/components/Footer.vue:
```diff
@@ -0,0 +1,59 @@
+        align="right"
+        class="primary"
+      >
+      © 2020 - Big Brothers Big Sisters Calgary
```
Try <span class-"my-auto"> © 2020 - Big Brothers Big Sisters Calgary </ span>

velvetarchangel replied:
did not seem to do the trick sadly :(. Did the best I can for now if it is an issue for the client will fix later.

ckthiessen replied:
style="margin: auto 0 auto 0" should work instead. "my-auto" is just a class with that style (but I can't get the vuetify CSS library working).

ckthiessen replied:
my-auto stands for margin-y-axis-auto. There is a similar one for centering stuff horizontally (mx-auto)

velvetarchangel committed:
BBSWA-38 Fixing the copyright text

Fixing copyright text to have consistency, added as a disabled button.
The text is grey right now but it is consistent. Minor fix will open
another ticket if its a big problem.
